phpcs.xml.dist

```diff
@@ -154,6 +154,7 @@
 
     <rule ref="SlevomatCodingStandard.Classes.SuperfluousInterfaceNaming">
         <exclude-pattern>src/EntityManagerInterface.php</exclude-pattern>
+        <exclude-pattern>src/Mapping/ClassMetadataFactoryInterface.php</exclude-pattern>
```

Comment:
Why was this excluded from phpcs checks? It's going to be a new interface that you can name without the superflous naming.

Reply:
`Doctrine\ORM\Mapping\ClassMetadataFactory` is already used (is the name of the current implementation).
Do you have a better proposal for the name of this interface?